pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -40,6 +41,17 @@ export const BlockDetails: FC<BlockDetailsProps> = ({ block, selectBlock }) => {
     }
   };
 
+  const fetchDisableAdminOperations = async () => {
+    const response = await fetch('/api/v1/flags');
+    const result = await response.json();
+    setDisableAdminOperations(result.data.disableAdminOperations);
+    console.log('disableAdminOperations', result.data.disableAdminOperations);
+  };
+
+  useEffect(() => {
+    fetchDisableAdminOperations();
+  }, []);
+
```

Comment:
Let's try to do this from /status/flags endpoint directly. Take a look at how we use it for `defaultStep` in PanelList : https://github.com/thanos-io/thanos/blob/main/pkg/ui/react-app/src/pages/graph/PanelList.tsx#L161

Reply:
its actually very convinent. i use this in blockDetails.tsx and it worked like a charm :
```
  const { response: flagsRes } = useFetch<FlagMap>(`/api/v1/status/flags`);
  const disableAdminOperations = flagsRes?.data?.['disable-admin-operations'] || false;
  ```